postgres/datadog_checks/postgres/config.py

```diff
@@ -44,6 +44,7 @@ def __init__(self, instance):
             raise ConfigurationError("Application name can include only ASCII characters: %s", self.application_name)
 
         self.query_timeout = int(instance.get('query_timeout', 5000))
+        self.idle_connection_timeout = int(instance.get('idle_connection_timeout', 60000))
```

Comment:
I don't think we need `int` here now that the conversion happens at the config level